db/migrate/20210422151926_create_targets.rb

```diff
@@ -0,0 +1,14 @@
+class CreateTargets < ActiveRecord::Migration[6.1]
+  def change
+    create_table :targets do |t|
+      t.string      :title,       null: false
+      t.float       :radius,      null: false
+      t.float       :latitude,    null: false
+      t.float       :longitude,   null: false
+      t.timestamps                null: false
+
+      t.references  :user,        null: false
```

Comment:
```suggestion
      t.references  :user, foreign_key: true, null: false
```
This adds a new annotation as well.